src/main/java/Todo.java

```diff
@@ -0,0 +1,17 @@
+public class Todo extends Task {
+    public Todo(String description) {
+        super(description);
+        this.taskType = TaskType.TODO;
+    }
+
+    @Override
+    public String toString() {
+        String str = "["

```

Comment:
I think either way works, but I did it in multiline in case I would have to append other strings in the future. To me, if it grows, it would be easier to maintain the coding style of 120chars and clearer to interpret.